internal/namespaces/baremetal/v1alpha1/testdata/test-create-server-errors-error-invalid-instance-type.stderr.golden

```diff
@@ -0,0 +1,4 @@
+Invalid value 'foobar' for arg 'type'
```

Comment:
errors should be tested with `CheckError`

golden files are usually used for UI